assets/admin/students/student-modal/index.js

```diff
@@ -11,6 +11,48 @@ import { __ } from '@wordpress/i18n';
 import CourseList from './course-list';
 import InputControl from '../../../blocks/editor-components/input-control';
 
+const AddButton = (
+	<Button className="sensei-student-modal__action--add" variant="primary">
+		{ __( 'Add to Course', 'sensei-lms' ) }
+	</Button>
+);
+
+const RemoveButton = (
+	<Button className="sensei-student-modal__action--remove">
+		{ __( 'Remove from Course', 'sensei-lms' ) }
+	</Button>
+);
+
+const ResetButton = (
+	<Button className="sensei-student-modal__action--remove">
+		{ __( 'Reset or Remove the student(s) progress', 'sensei-lms' ) }
```

Comment:
This should be "Reset or Remove Progress" as per the [designs](https://www.figma.com/file/m9w7QkHLoNUgxFsp7jH75B/Importing-Users-%26-Course-Grouping?node-id=1300%3A4800).